book/forms.rst

```diff
@@ -625,6 +625,21 @@ Now the form will skip your validation constraints. It will still validate
 basic integrity constraints, such as checking whether an uploaded file was too
 large or whether you tried to submit text in a number field.
 
+Handling Additional Fields
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 2.6
+    The ``allow_extra_fields`` option was introduced in Symfony 2.6.
+
+Usually, the Form component will complain when your submitted form contains
+fields that were not configured before (for example, when fields are added
+using JavaScript or when a page contains several forms without a name).
```

Comment:
Not sure how often you really want to submit multiple nameless forms to the same endpoint (this was a reason given in symfony/symfony#8133). However, you will probably notice when you need this option. So I decided to remove this part.